source/shared/cpp/ObjectModel/Container.cpp

```diff
@@ -108,6 +124,13 @@ std::shared_ptr<Container> Container::Deserialize(const Json::Value& value)
     // Parse Items
     auto cardElements = ParseUtil::GetElementCollection<BaseCardElement>(value, AdaptiveCardSchemaKey::Items, Container::CardElementParsers, true);
     container->m_items = std::move(cardElements);
+
+    Json::Value selectActionValue = ParseUtil::ExtractJsonValue(value, AdaptiveCardSchemaKey::SelectAction, false);
+    if (!selectActionValue.empty())
+    {
+        container->SetSelectAction(ParseUtil::GetActionFromJsonValue<BaseActionElement>(selectActionValue, BaseCardElement::ActionParsers));
+    }
```

Comment:
I can do that for the ActionParsers since there is no plan for specific elements to support different action types, but I believe the CardElementParsers should remain in place, as not all of them are the same.